### DISCUSSION TITLE
[Spring] 테스트 실행 시 "Could not autowire, No beans of 'QueryCounter' type found." 에러

### DISCUSSION THREAD

**Discussion by SohnEugene** [wafflestudio/seminar-2023#167] [Q&A] [answered]
안녕하세요. 테스트가 실행이 안 되는 문제가 있어서 질문 남깁니다.
![image](https://github.com/wafflestudio/seminar-2023/assets/102849934/d93222d2-a3bc-42e4-b4fd-55a6ad198e17)
이와 같이 테스트를 실행하면 빈이 자동으로 생성되지 않는 오류가 떠서 테스트 실행을 하지 못하고 있습니다. 인터넷에서 찾아보면 인텔리제이 자체의 오류라고 하는데, 몇 가지 해결 방법을 실행해봐도 해결이 되지 않습니다. 그에 더해 잘 됐던 week0 과제 테스트 코드에서도 같은 오류가 뜨는데 혹시 무엇이 문제인지 알 수 있을까요?

**Comment by SohnEugene** [comment, marked as answer]
에러 메세지 확인하고 songRepository에 작성해놓은 임시 코드 삭제하고 나니 테스트도 runApplication도 제대로 작동했습니다. 잘 확인하지 않고 질문드려 죄송합니다.

**Reply by PFCJeong** [reply]
안녕하세요~ Could not autowire. 요 경고 메시지는 인텔리제이 오류가 맞습니다. 실행하는데는 지장이 없습니다.
